gm4_vecto_shamir/data/gm4_vecto_shamir/functions/vehicle/spawn_boat.mcfunction

```diff
@@ -0,0 +1,20 @@
+#run from vehicle/check_water and active
+#@s = player with shamir equipped, sneaking above water OR ice
+
+tag @s add gm4_ml_vecto_active
+
+# spawn boat
+execute if entity @s[y_rotation=-45..45] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_ml_vecto_vehicle"],Rotation:[0f,0f]}
+execute if entity @s[y_rotation=45..135] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_ml_vecto_vehicle"],Rotation:[90f,0f]}
+execute if entity @s[y_rotation=135..225] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_ml_vecto_vehicle"],Rotation:[180f,0f]}
+execute if entity @s[y_rotation=-135..-45] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_ml_vecto_vehicle"],Rotation:[270f,0f]}
```

Comment:
```suggestion
execute if entity @s[y_rotation=-45..45] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_vecto_vehicle"],Rotation:[0f,0f]}
execute if entity @s[y_rotation=45..135] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_vecto_vehicle"],Rotation:[90f,0f]}
execute if entity @s[y_rotation=135..225] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_vecto_vehicle"],Rotation:[180f,0f]}
execute if entity @s[y_rotation=-135..-45] run summon minecraft:boat ~ ~ ~ {Invulnerable:1b,Tags:["gm4_vecto_vehicle"],Rotation:[270f,0f]}
```
Shamirs should not use the `ml_` prefix. This change needs to be made in multiple sections of code throughout the module.